File: rudolph/src/main/java/cn/wzbos/android/rudolph/OnRouteListener.java
package cn.wzbos.android.rudolph;

import cn.wzbos.android.rudolph.router.Router;

/**
 * Created by wuzongbo on 2017/4/12.
 */
public interface OnRouteListener {

    void onSuccess(Router<?> router);

    void onError(Router<?> router, Throwable throwable);
}

File: rudolph/src/main/java/cn/wzbos/android/rudolph/OnRouteListener.kt
package cn.wzbos.android.rudolph

import cn.wzbos.android.rudolph.router.Router

/**
 * Created by wuzongbo on 2017/4/12.
 */
interface OnRouteListener {
    fun onSuccess(router: Router<*>?)
    fun onError(router: Router<*>?, throwable: Throwable?)
}